test/end-to-end.js

```diff
@@ -970,6 +977,58 @@ describe('end-to-end', function() {
       });
     });
 
+    describe('error handling in dev/prod', function() {
+      it('includes sensitive error details in development mode', function(done) {
+        var app = loopback({ localRegistry: true, loadBuiltinModels: true });
+        var bootOptions = {
+          appRootDir: SANDBOX,
+          env: 'development'
+        };
+        boot(app, bootOptions, function(err) {
+          if (err) return done(err);
+          app.listen(0, function() {
+            request(app)
+              .get('/url-does-not-exist')
+              .expect(404)
+              .end(function(err, res) {
+                app.use(function(err, res) {
+                  var responseBody = JSON.stringify(res.body);
+                  console.log(responseBody);
```

Comment:
Forgotten `console.log`. If you find it useful to have a log around, then please use `debug` instead.

In case `debug` is not defined in this file yet: in tests, we usually define `debug` at the top of the test file this way:

```js
var debug = require('debug')('test');
```